std/algorithm/comparison.d

```diff
@@ -1565,3 +1565,184 @@ unittest
                 2, "two",
                 ));
 }
+
+// TODO Why does this return $(D false) when $(D T) is a $(D string) or array: isImplicitlyConvertible!(bool, T);
+enum implicitlyConvertsToBool(T) = isImplicitlyConvertible!(bool, T) || isArray!T;
```

Comment:
So this is public and undocumented? Also it's a misnomer - arrays are not convertible to bool, they're just if-testable.